lib/logstash/outputs/elasticsearch/elasticsearch-template.json

```diff
@@ -5,88 +5,37 @@
   },
   "mappings" : {
     "_default_" : {
-      "_all" : {"enabled" : true, "omit_norms" : true},
```

Comment:
If I recall correctly, when I tested a _new_ 5.0 compatible template (rather than just hoping the existing template would be adapted on the fly) it didn't work with `"norms" : false`.  I got some error or other.